openvdb_houdini/ParmFactory.cc

```diff
@@ -1316,6 +1317,12 @@ OpFactory::~OpFactory()
     if (mImpl->mInvisible) {
         mImpl->mTable->addOpHidden(mImpl->mName.c_str());
     }
+
+    // hide other nodes in invisible name list
+
+    for (const auto& name : mImpl->mInvisibleNames) {
+        mImpl->mTable->addOpHidden(name.c_str());
+    }
```

Comment:
@pcucka - I could see two main options as to how to provide node names to hide to the operator table. This option (the one I went with) simply stores a vector of names to be hidden on the factory implementation class. The alternative I considered was to add a protected non-const OpFactory::table() method so as to be able to modify it directly from the derived class.

I don't really have a preference here though, so if you'd prefer the other way of doing this or have any other ideas, I'm happy to do this differently.